src/BannerWithLink/index.tsx

```diff
@@ -91,37 +63,25 @@ const BannerWithLink: React.FC<Props> = (props: Props) => {
                         </p>
                     ) : null}
 
-                    <LinkButton
-                        href={buttonUrl}
-                        css={styles.primaryButton}
-                        onClick={() =>
-                            trackClick({
-                                internalButtonId: 0,
-                                ophanComponentId: ophanComponentId as string,
-                            })
-                        }
-                    >
-                        {buttonText}
-                    </LinkButton>
+                    <PrimaryCtaButton
+                        buttonText={buttonText as string}
+                        buttonUrl={buttonUrl as string}
+                        showPaymentIcons={false}
+                        ophanComponentId={ophanComponentId as string}
```

Comment:
Again, in the interest of keeping interfaces simple, instead of passing through the `ophanComponentId` only to pass it to the `trackClick` callback what if we collapsed this and did the wiring outside? So drop the `ophanComponentId` arg and update `trackClick` to be `() => trackClick(ophanComponentId)` or similar? Actually it looks like `trackClick` also takes the `internalButtonId` which is defined in the `PrimaryCtaButton` component so it'd have to be provided, so the callback passed here would be something more like `(internalButtonId) => trackClick({ internalButtonId, ophanComponentId })`